docs/user/dashboard/lens-end-to-end/lens-end-to-end.asciidoc

```diff
@@ -0,0 +1,506 @@
+[float]
+[[lens-end-to-end]]
+== Try it: Analyze website data on a dashboard
+
+You have collected data about your webiste, and you're ready to find the answers to some key questions. 
+
+When you're finished creating the dashboard, you'll be able to answer the following questions:
+
+* How many users have visited my website?
+* What’s the distribution of visitors by operating system?
+* What’s the ratio of code responses over time?
+* What’s the average bytes transfer per day? How does it relate to the number of visitors?
+* What's the percentage of small vs big files transfer?
+* How does website traffic look like hour by hour?
+* Where does the traffic come from for top three countries?
+
+[discrete]
+[[add-the-sample-web-logs-data]]
+=== Add the web logs data
+
+To analyze the website traffic data, add the data set to {kib}.
+
+. From the {kib} *Home* page, click *Try our sample data*.
+
+. From *Sample web logs*, click *Add data*.
+
+[discrete]
+[[end-to-end-create-the-dashboard]]
+=== Create the dashboard
+
+Open the dashboard, then open the visualization builder. 
+
+. Open the main menu, click *Dashboard*.
+
+. Click *Create dashboard*.
```

Comment:
Can we combine these two sections into something shorter? I'm not sure it really needs bullet points.